.buildkite/pipeline.yaml

```diff
@@ -31,3 +31,16 @@ steps:
       - docker#v3.0.1:
           image: "node:10"
           propagate-environment: true
+
+  - label: ":hammer: package"
+    command:
+      - "echo '--- Fetching Dependencies'"
+      - "./scripts/fetch-develop.deps.sh --depth 1"
+      - "yarn install"
+      - "echo '+++ Packaging'"
+      - "./scripts/ci_package.sh"
+    #branches: "travis/test-build"
```

Comment:
Remove?

Reply:
ah, sorry, I thought I got in and changed that to `develop` before you looked :p

Reply:
(and yep, I'll remove the ones travis added)